zxing-android-embedded/src/com/journeyapps/barcodescanner/ViewfinderView.java

```diff
@@ -131,84 +133,91 @@ protected void refreshSizes() {
             return;
         }
         Rect framingRect = cameraPreview.getFramingRect();
-        Size previewSize = cameraPreview.getPreviewSize();
-        if (framingRect != null && previewSize != null) {
+        Rect previewFramingRect = cameraPreview.getPreviewFramingRect();
+        if (framingRect != null && previewFramingRect != null)
+        {
```

Comment:
Please consider to not change the codestyle.
Unfortunately the maintainer is not active, otherwise I would make a pull request to check for right code style

Btw, thank you for the feature. Btw a screenshot would be cool, to see immediate what is introduced

Reply:
![IMG_20201113_233240 120](https://user-images.githubusercontent.com/40764403/99105513-ab45bc00-2608-11eb-8f98-3efb6aaee503.png)

Really sorry for this late. I missed this. Could you please check if we can have this kind of border as border always helps in background setting.